src/main/java/edu/kis/vh/nursery/list/Node.java

```diff
@@ -1,12 +1,41 @@
 package edu.kis.vh.nursery.list;
 
+/**
+ *  protected Node prev - previous list's node
+ *  protected Node next - next list's node
+ *	private final int value - value of node
+ *	public Node(int i) - constructor to initialize Node
+ *  public int getValue(), 	public Node getPrev(), public void setPrev(Node prev), public void setNext(Node next)... - methods to access and update the value
```

Comment:
This is not a proper javadoc. Each non-private element should be commented separately.